content/en/docs/_index.md

```diff
@@ -4,28 +4,9 @@ linkTitle: Docs
 menu: { main: { weight: 10 } }
 ---
 
-OpenTelemetry is a set of APIs, SDKs, tooling and integrations that are designed
-for the creation and management of _telemetry data_ such as traces, metrics, and
-logs. The OpenTelemetry documentation is intended to broadly cover key terms,
-concepts, and instructions on how to use OpenTelemetry in your software.
+OpenTelemetry, also known as OTel for short, is a vendor-neutral open-source [Observability](concepts/observability-primer/#what-is-observability) framework for instrumenting, generating, collecting, and exporting telemetry data ([traces](concepts/observability-primer/#distributed-traces), [metrics](concepts/observability-primer/#reliability--metrics), [logs](concepts/observability-primer/#logs)). As an industry-standard it is [natively supported by a number of vendors](/vendors)
 
-![Implementation of the OpenTelemetry Reference Architecture](https://raw.github.com/open-telemetry/opentelemetry.io/main/iconography/Reference_Architecture.svg)
+![OpenTelemetry Reference Architecture](/img/otel_diagram.png)
```

Comment:
a suggestion, could be solved in a separate PR: it's good practice to provide a description of the picture in the alt text so that people who don't see pictures still get some context.